pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -265,6 +286,11 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 		return nil, fmt.Errorf("at least one bind plugin is needed")
 	}
 
+	f.preemptHandle = &preemptHandle{
+		extenders:     options.extenders,
+		PodNominator:  options.podNominator,
+		PluginsRunner: f,
+	}
```

Comment:
I would put this right after the instantiation of the framework to avoid returning a not fully initialized framework at line 219.